test_data/layer_collection_defs/test_collection.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It would be nice if you could replicate the dataset list of our test data with the new layer stuff.

Reply:
Yeah we could, for now, have two collections: datasets and derived layers. then we could (in theory) get rid of the dataset listing. As long as we add all datasets also as layers...

I could either create an additional json file for each dataset or code something in the `new_with_data` method of the contexts to automatically create a layer for each dataset that was loaded. What would you prefer.

However it will remain a problem that newly created datasets e.g. via upload, won't show up in the layer listing. Maybe we need to modify the dataset creation and add a step where *always* a corresponding layer is created when a dataset is created. Or even better: create the layers on the fly in a dynamic dataset category...

Reply:
Well, if you could do this for the test datasets, this could work.
Maybe we need a flag then to prevent this.

Can't you create a workflow at the same time you create a dataset after upload?

Reply:
I added the functionality to create layers from the test datasets: a263c1de4bc11ac08b2333b02b326face3804a31

as for the other occasions where datsets are created I guess we should only create layers automatically when we have some rights management for the layers, because otherwise anyone can see anyone else's uploads as layers